src/ReactWrapper.jsx

```diff
@@ -885,12 +885,23 @@ class ReactWrapper {
   }
 
   /**
-   * Returns true if the current wrapper has no nodes. False otherwise.
+   * Delegates to exists()
    *
    * @returns {boolean}
    */
   isEmpty() {
-    return this.length === 0;
+    // eslint-disable-next-line no-console
+    console.warn('Deprecated method isEmpty() called, use exists() instead.');
```

Comment:
Actually I'm wondering if we should hold off on the deprecation notice in-code for now, until we've been able to make PRs into things like `chai-enzyme` and others - ie, give a whole release before adding the warning message in?